docs/start.rst

```diff
@@ -61,6 +61,53 @@ To enable float accuracy as opposed to double, run CMake with the appropriate fl
 
     qc/ $ cd qrack/build && cmake -DENABLE_COMPLEX8=ON ..
 
+Using the API
```

Comment:
Include some more basic examples of using the factory to create QEngines?

Update the section above this to remove the note about ENABLE_OPENCL from the vm6502q compilation options.